v0/pipeline.json

```diff
@@ -56513,6 +56513,15 @@
                   "items" : {
                     "type" : "string"
                   }
+                },
+                "privileged" : {
+                  "oneOf" : [ {
+                    "type" : "boolean"
+                  }, {
+                    "type" : "string",
+                    "pattern" : "^<\\+input>((\\.)((executionInput\\(\\))|(allowedValues|default|regex)\\(.+?\\)))*$",
```

Comment:
@piyush-bhuwalka Have you tested, whether this expression will work with runtime inputs?

Reply:
Yes @ivan-83 I have tested it with runtime input. Its working fine